website/vercel.json

```diff
@@ -1201,6 +1201,36 @@
     {
       "source": "/getting-started/tutorials/customer-support-tool/application-settings",
       "destination": "/getting-started/tutorials/start-building"
+    },
+    {
+      "source": "/widget-reference/file-picker",
+      "destination": "/reference/widgets/filepicker",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/map",
+      "destination": "/reference/widgets/maps",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/dropdown",
+      "destination": "/reference/widgets/select",
+      "permanent": true
+    },
+    {
+      "source": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud?q=error",
```

Comment:
reference/datasources - Is. a wrong URL. Let's remove it from the index.

Reply:
`/reference/datasources/querying-mongodb/whitelist-mongodb-cloud` should be redirected to - https://studio3t.com/knowledge-base/articles/mongodb-atlas-login-ip-whitelisting/#how-to-whitelist-ip-addresses-on-mongodb-atlas

Reply:
Only one redirect rule is to be added here for the whitelist.

Reply:
updated